src/readme.js

```diff
@@ -63,8 +59,36 @@ const buildReadmeContent = async (context, templateName) => {
   })
 }
 
+/**
+ * Get path to the readme template
+ *
+ * @param {string} availableTemplate
+ * @param {string} customTemplate
+ */
+const getReadmeTemplatePath = args => {
+  const spinner = ora('Resolving README template path').start()
+
+  const { template: availableTemplate, path: customTemplate } = args
+
+  const templatePath =
+    customTemplate ||
+    path.resolve(__dirname, `../templates/${availableTemplate}.md`)
+
+  try {
+    fs.lstatSync(templatePath).isFile()
```

Comment:
This will still pass the cases for path with empty string or space-only (would use the default path if so), like these:
```
npx readme-md-generator -p -y

npx readme-md-generator -p  -y

npx readme-md-generator -p   -y 
```
We could add the below line to solve:
```js
customTemplate !== undefined && fs.lstatSync(customTemplate).isFile()
```

What do you think of these cases? If okay, I would handle it, or else we could finish this PR.

Reply:
You're right, no error is displayed if path is empty 👍 I made a small changes in the last commit to fix this. Is that ok for you ?

Reply:
Worked well, very nice. Thank you.